src/components/Common/ButtonV2.tsx

```diff
@@ -101,7 +101,7 @@ export const buttonStyles = ({
   shadow = !ghost,
 }: ButtonStyleProps) => {
   return classNames(
-    "inline-flex h-min cursor-pointer items-center justify-center gap-2 whitespace-pre font-medium outline-offset-1 transition-all duration-200 ease-in-out disabled:cursor-not-allowed disabled:bg-secondary-200 disabled:text-secondary-500",
+    "inline-flex h-min cursor-pointer items-center justify-center gap-2 whitespace-pre mt-2 font-medium outline-offset-1 transition-all duration-200 ease-in-out disabled:cursor-not-allowed disabled:bg-secondary-200 disabled:text-secondary-500",
```

Comment:
ButtonV2 is widely used - make sure this doesn't cause any issues elsewhere. Instead of modifying it here, modify wherever the specific issue is occurring.

Reply:
@gurramkarthiknetha We are switching over the Shadcn, gradually phasing out ButtonV2, so let's not modify here anyways.